build.sbt

```diff
@@ -292,6 +292,24 @@ lazy val scioBigtable: Project = Project(
   scioTest % "it"
 ).configs(IntegrationTest)
 
+lazy val scioElsticsearch: Project = Project(
+  "scio-elasticsearch",
+  file("scio-elasticsearch")
+).settings(
+  commonSettings ++ itSettings,
+  description := "Scio add-on for Writing to Elasticsearch Cluster",
+  libraryDependencies ++= Seq(
+    "org.apache.beam" % "beam-sdks-java-io-google-cloud-platform" % beamVersion,
+    "com.google.auto.value" % "auto-value" % "1.2",
+    "joda-time" % "joda-time" % jodaTimeVersion,
+    "org.elasticsearch" % "elasticsearch" % "2.1.0",
```

Comment:
@lmayumi well honestly not sure ... is providing both versions out of question? 😄 looking at some sort of maven stats ... both 2.X and 5.X show a lot of downloads... I'm also a little bit biased cause i have most of the stuff in 2.X